build.py

```diff
@@ -1726,6 +1768,12 @@ def core_build(
     cmake_script.blankln()
 
 
+def tensorrtllm_prebuild(cmake_script):
+    # Export the TRT_ROOT environment variable
+    cmake_script.cmd("export TRT_ROOT=/usr/local/tensorrt")
```

Comment:
This can be simplified if we are replacing TRT8 with TRT9 in-place.

Reply:
Synced offline that we will improve the TRT9 installation in the future as right now it requires to modify the build script in the TRT-LLM repo.